src/features/ModelPlan/ModelToOperations/_components/Table/index.tsx

```diff
@@ -58,16 +66,6 @@ const MTOTable = () => {
     ]
   });
 
-  const {
-    data: queryData,
-    loading,
-    error
-  } = useGetModelToOperationsMatrixQuery({
-    variables: {
-      id: modelID
-    }
-  });
-
```

Comment:
the gql call is pushed up to the parent component and `queryData` is passed down as a prop. see above comment